tests/_core/nodes.yaml

```diff
@@ -7,3 +7,53 @@ chapters:
     method: GET
```

Comment:
This file should be moved under `tests/nodes/` and split into `info.yaml` and `stats.yaml`